Makefile

```diff
@@ -3,11 +3,24 @@ test: venv
 	venv/bin/pre-commit install
 	venv/bin/pre-commit run --all-files
 
+.PHONY: builddeb
+builddeb: autoversion
+	dpkg-buildpackage -us -uc
+
+.PHONY: autoversion
+autoversion:
+	date +%Y.%m.%d.%H.%M-git`git rev-list -n1 HEAD | cut -b1-8` > .version
+	rm -f debian/changelog
+	DEBFULLNAME="Open Computing Facility" DEBEMAIL="help@ocf.berkeley.edu" VISUAL=true \
```

Comment:
sm+packages@ocf.berkeley.edu perhaps?

Reply:
This was the same as in approve and the email we use for ocflib in the setup.py.

Reply:
perhaps we should change it elsewhere as well, I don't have particularly strong opinions except that help@ seems inappropriate, being a customer service queue.

Reply:
Yeah, it did seem weird. I'm okay changing it if we do so elsewhere.